DESCRIPTION

```diff
@@ -1,4 +1,4 @@
-Package: rphg2
```

Comment:
Since we are going with phg, should this be rphg?

Reply:
This will have to be `rPHG2` to avoid package collisions with the already existing `rPHG`.